synapse/cores/ram.py

```diff
@@ -175,6 +180,25 @@ def _getSizeByProp(self, prop, valu=None, mintime=None, maxtime=None):
 
         return len(rows)
 
+    def _getCoreType(self):
+        return 'ram'
+
+    def _getBlobValu(self, key):
+        ret = self._blob_store.get(key)
+        return ret
+
+    def _setBlobValu(self, key, valu):
+        mesg = 'Setting blob value in ram cortex [{}]. Not a persistent action.'.format(self.myfo[0])
+        self.log(logging.WARNING, mesg=mesg, key=key, valu=valu)
```

Comment:
if we plumb this into the savefile, we're now plumbing the storage layer data back up into the eventbus (which we're kind of already doing with the log here).  since we've only ever talked about using savefile for ram cortexes, do we want to pull this down into the ram cortex so that the savefile concept is specific to the ram cortex?

currently, you can use the savefile to save events off of any cortex type.  I worry about adding the complexity of savefile now and then having to turn around and deal with that when separating out storage backings from the Cortex class itself.